packages/components/image-viewer/src/image-viewer.vue

```diff
@@ -37,6 +37,15 @@
               </el-icon>
             </span>
           </template>
+          <span :class="[ns.e('btn'), ns.e('progress')]">
```

Comment:
```suggestion
          <div :class="[ns.e('btn'), ns.e('progress')]">
```
It is better to change to div tag here. If slot adds div tag inside span tag, it will cause structural error.